src/Files.App/ViewModels/Previews/MediaPreviewViewModel.cs

```diff
@@ -21,23 +21,28 @@ public MediaSource Source
 
 		public MediaPreviewViewModel(ListedItem item) : base(item) { }
 
-		public static bool ContainsExtension(string extension) => extension
-			is ".mp4" or ".webm" or ".ogg" or ".mov" or ".qt" or ".mp4" or ".m4v" // Video
-			or ".mp4v" or ".3g2" or ".3gp2" or ".3gp" or ".3gpp" or ".mkv"        // Video
-			or ".mp3" or ".m4a" or ".wav" or ".wma" or ".aac" or ".adt" or ".adts" or ".cda" or ".flac"; // Audio
+		public static bool ContainsExtension(string extension)
```

Comment:
I think we have a an existing helper that does this, can we combine the two and start using the helper? Feel free to add a TODO.